src/CustomItem.cs

```diff
@@ -11,12 +16,13 @@ public class CustomItem
 	public string Name;
 	public string Description;
 
-	private readonly GameObject ItemPrefab;
-	public GameObject WorldObject {get; private set;}
+	public GameObject ItemPrefab { get; private set; }
 
 	public InventoryItemSpec ItemSpec => ShopData.item;
 	// public ShelfItem ShelfComponent => ShopData.shelfItem;
```

Comment:
commented code, can we get rid of this?

Reply:
Yeah, a lot of commented out code can be cleaned up - I'll take one more pass at that